### ISSUE TITLE
Clarify when configuration is ready and valid

### ISSUE THREAD

**Issue by jkva** [coyote-team/coyote-wp#61]

**Comment by jkva** [comment]
It is indicated to the user when:

* an invalid token / endpoint combination is used ("unable to load Coyote profile")
* a valid token / endpoint combination is used ("username: foo bar")
* organizations are available ("please select an organization...")
* organizations are switched ("this requires a fresh import...")

That should be sufficient.

**Comment by jkva** [comment]
Increase visual clarity for error / succes indication.

**Comment by robbinrietdijk** [comment]
The flow of settings up the plugin has been improved. The only notice that isn't active at the moment is when an invalid combination of API key and endpoint is used.

**Comment by jkva** [comment]
Since this combination resolves either way in an "couldn't load profile" notification, this has been addressed.
